src/redisenterprise/azext_redisenterprise/generated/commands.py

```diff
@@ -0,0 +1,62 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    from azext_redisenterprise.generated._client_factory import cf_operation_status
+
+    redisenterprise_operation_status = CliCommandType(
+        operations_tmpl='azext_redisenterprise.vendored_sdks.redisenterprise.operations._operations_status_operations#OperationsStatusOperations.{}',
+        client_factory=cf_operation_status,
+    )
+    with self.command_group(
+            'redisenterprise operation-status', redisenterprise_operation_status, client_factory=cf_operation_status
```

Comment:
could we design the command with something like `az redis-enterprise xxxx`?

Reply:
What is the motivation for renaming all of the commands?  I originally went without a hyphen in the name because I saw in the core repo that there are no hyphens in the names: https://github.com/Azure/azure-cli/tree/dev/src/azure-cli/azure/cli/command_modules

Reply:
I strongly prefer to leave as `az redisenterprise xxxx`

Reply:
Actually there is CLI command guideline for command naming https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior

Reply:
I believe I am following the command naming guidelines for all of my commands.  "az redisenterprise" is the module, not the command.  See the example "az keyvault" on your link: https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior